File: benchmarks/sum/sum.py
import sys
import time

N = int(sys.argv[1])

t0 = time.perf_counter()

sum_ = 0
while N > 0:
  sum_ += N
  N -= 1

print("Total sum: " + str(sum_))

t1 = time.perf_counter()

print("Execution took (ms) " + str((t1 - t0) * 1000))

File: benchmarks/sum/sum.lua

local t0 = os.clock()
local N = tonumber(arg[1])

local sum = 0
while N > 0 do
    sum = sum + N
    N = N - 1
end

print("Total sum: " .. sum)

local t1 = os.clock()

print("Execution took (ms) " .. ((t1 - t0) * 1000))